BioArchLinux/minimap2/lilac.yaml

```diff
@@ -12,3 +12,18 @@ update_on:
     github: lh3/minimap2
     use_max_tag: true
     prefix: 'v'
+  - source: alpm
+    alpm: zlib
+    repo: core
+    provided: libz.so
+    strip_release: true
+  - source: alpm
+    alpm: glibc
+    repo: core
+    provided: libm.so
+    strip_release: true
+  - source: alpm
+    alpm: glibc
+    repo: core
+    provided: libc.so
+    strip_release: true
```

Comment:
The `glibc` package doesn't list any libraries in its `provides` array, so this will not work for `libm.so` and `libc.so`. These libraries however are unlikely to get soname changes so it's safe to omit them. I usually don't list the `glibc` and `gcc-libs` libraries in the rebuild configs for this reason, even though it would be possible using the `alpmfiles` source.